tests/functional/spec/features/units.spec.js

```diff
@@ -32,4 +32,12 @@ describe("Units", () => {
     expect($(SubmitPage.durationHour()).getText()).to.equal("6 awr");
     expect($(SubmitPage.durationYear()).getText()).to.equal("20 mlynedd");
   });
+
+  it("Given we open a questionnaire with unit labels, when the label is highlighted by the tooltip, then the long unit label should be displayed.", () => {
+    browser.openQuestionnaire("test_unit_patterns.json", { language: "en" });
+    expect($("body").getHTML()).to.contain('title="centimeters"');
+    expect($("body").getHTML()).to.contain('title="meters"');
+    expect($("body").getHTML()).to.contain('title="kilometers"');
+    expect($("body").getHTML()).to.contain('title="miles"');
```

Comment:
Minor, but we should avoid checking the entire html where possible. For this, we can get the value of the title explicitly, e.g:
```js
    expect($(SetLengthUnitsBlockPage.centimetres()).getAttribute("title")).to.equal("centimeters");
```